src/main/java/bio/terra/janitor/service/stackdriver/StackdriverExporter.java

```diff
@@ -26,24 +21,14 @@ public StackdriverExporter(StackdriverConfiguration stackdriverConfiguration) {
   }
 
   public void initialize() {
-    logger.info("Stackdriver enabled: {}.", stackdriverConfiguration.isEnabled());
+    logger.info("Stackdriver stats enabled: {}.", stackdriverConfiguration.isEnabled());
     if (!stackdriverConfiguration.isEnabled()) {
       return;
     }
     try {
       StackdriverStatsExporter.createAndRegister();
-      StackdriverTraceExporter.createAndRegister(StackdriverTraceConfiguration.builder().build());
     } catch (IOException e) {
-      throw new RuntimeException("Unable to initialize Stackdriver exporting.", e);
+      throw new RuntimeException("Unable to initialize Stackdriver stats exporting.", e);
```

Comment:
do you also switch to `logger.error` instead of exception?

Reply:
thanks, done.